WordPress/src/main/res/values/attrs.xml

```diff
@@ -107,4 +107,17 @@
         <attr name="wpTextDialogSubtitle" format="string" />
     </declare-styleable>
 
+    <!--
+        WPLoginInputRow
+    -->
+    <declare-styleable name="wpLoginInputRow">
+        <attr name="wpIconDrawable" format="reference"/>
+        <attr name="wpIconContentDescription" format="string"/>
+        <attr name="android:inputType" />
+        <attr name="android:hint" />
+        <attr name="android:imeOptions" />
+        <attr name="passwordToggleEnabled" />
+        <attr name="passwordToggleTint" />
+    </declare-styleable>
```

Comment:
Not really, the formats get inherited from the design support lib's definitions.